src/sentry/static/sentry/app/components/circleIndicator.tsx

```diff
@@ -2,15 +2,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 
-const getBackgroundColor = p => {
+type Props = {
+  enabled: boolean;
+  size: number;
+  color: string;
+};
+
+type StyledProps = Props & {theme?: any};
```

Comment:
I'd like to come up with a cleaner solution for these, but I think this is what we'll have to do for now 🤷‍♂

Reply:
@EvanPurkhiser What if we made a `themeProp` type that is just `{theme?: any}` for now? That way we can just update it once as long as everything is importing that one type

Reply:
Well what would probably be better is for us to actually just use our theme.tsx stuff

Reply:
@EvanPurkhiser You mean create a type for it in that file?

Reply:
Yeah something like that

Reply:
Don't worry about it for now